sessions/publisher.go

```diff
@@ -36,6 +36,11 @@ func (p *publisher) publish(sessions []*Session) error {
 		// log every minute
 		return nil
 	}
+	nrs, rs := p.config.NotifyReleaseStages, p.config.ReleaseStage
+	if nrs != nil && !contains(nrs, rs) {
```

Comment:
We should add a test and check that delivers when release stage is "" and notify release stages is nil

Reply:
Added a test for release stage being `""`. Most of the integration tests already verify that `NotifyReleaseStages` being nil still results in the session payload being published.